pkg/services/rpcsrv/client_test.go

```diff
@@ -2447,8 +2447,11 @@ func TestClient_GetStorageHistoric(t *testing.T) {
 }
 
 func TestClient_GetVersion_Hardforks(t *testing.T) {
-	_, _, httpSrv := initServerWithInMemoryChain(t)
-
+	_, _, httpSrv := initClearServerWithCustomConfig(t, func(cfg *config.Config) {
+		cfg.ProtocolConfiguration.Hardforks = map[string]uint32{
+			"Aspidochelone": 25,
+		}
+	})
```

Comment:
`initClearServerWithCustomConfig` should be kept, but uou don't need this Aspid-related change. Fix the check down below in accordance with unit test forks config.

Reply:
Yes, I think it's OK for this particular test, because it's the test purpose to check enabled hardforks.